cli/internal/helm/helm_test.go

```diff
@@ -107,3 +119,148 @@ func TestMergeMaps(t *testing.T) {
 		})
 	}
 }
+
+func TestHelmApply(t *testing.T) {
+	cliVersion := semver.NewFromInt(2, 11, 0, "")
+	csp := cloudprovider.AWS // using AWS since it has an additional chart: aws-load-balancer-controller
+	microserviceCharts := []string{
+		"constellation-services",
+		"constellation-operators",
+		"constellation-csi",
+	}
+	testCases := map[string]struct {
+		clusterMicroServiceVersion string
+		expectedActions            []string
+		clusterCertManagerVersion  *string
+		clusterAWSLBVersion        *string
+		expectNoUpgrade            bool
```

Comment:
```suggestion
		expectUpgrade            bool
```

Less double negation in our heads, more better :D

Reply:
I know ;) I only did it for convenience, otherwise we need to write expectUpgrade in every upgrade which is redundant, since `expectedActions`  makes that implicitly clear. Any suggestion?

Reply:
I'd just add the `expectUpgrade: true,`.